satellite/console/service.go

```diff
@@ -230,7 +236,7 @@ func (s *Service) ResetPassword(ctx context.Context, resetPasswordToken, passwor
 		return err
 	}
 
-	if time.Since(claims.Expiration) > 0 {
+	if time.Since(token.CreatedAt.Add(tokenExpirationTime)) > 0 {
```

Comment:
`time.Since(token.CreatedAt) < tokenExpirationTime` or `>` whichever the appropriate comparison operator is.